mojaloop/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-description: Mojaloop Helm chart for Kubernetes
+description: Helm chart for Mojaloop. This is base Dwolla code that works with postgres backend.
```

Comment:
This is not correct. I would suggest reverting back to "Mojaloop Helm chart for Kubernetes". The comment about Dwolla is of no concern to people who will be using these charts. It is only applicable to the internal OSS team.